charts/latest/azuredisk-csi-driver/values.yaml

```diff
@@ -68,7 +70,7 @@ windows:
       repository: mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar
       tag: v2.0.1
       pullPolicy: IfNotPresent
-    
```

Comment:
remove empty line

Reply:
its a trailing whitespace that i removed (editor has done it). do you suggest me to remove the line altogether?

Reply:
no, thanks.